src/amp-story-player/amp-story-player-impl.js

```diff
@@ -678,7 +689,18 @@ export class AmpStoryPlayer {
    * @private
    */
   previous_() {
-    if (this.currentIdx_ - 1 < 0) {
+    if (
```

Comment:
I think this can be changed to
`!this.circularWrapping_() || !this.isIndexOutofBounds_(this.currentIdx_ -1)`
which is the logical equivalent of not (the second condition at line 699). Or you could just remove this line. I feel like it is a bit unnecessary because there's only one condition in which you want to call `go` and there are three conditions, ie. the remaining other conditions, where you don't call anything/return.